### PR TITLE
Add set_guest_wifi_password description

### PR DESCRIPTION
**Pull request by sti0**
Update docs for PR https://github.com/home-assistant/core/pull/62892

## Proposed change

- Add documentation for new service `set_guest_wifi_password` (https://github.com/home-assistant/core/pull/62892)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/62892
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fritz.markdown [hunk 1]
```diff
@@ -56,7 +56,7 @@ Reboot the router.
 
 | Service data attribute | Optional | Description                                                                                                    |
 | ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
-| `entity_id`            | no       | Only act on a specific  router                                                                                 |
+| `device_id`            | no       | Only act on a specific  router                                                                                 |
```
Comment (frenck) [inline source/_integrations/fritz.markdown:59]: How is this related?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fritz.markdown (hunk 1 above)
Comment (sti0) [reply source/_integrations/fritz.markdown:59]: Current services are using `device_id` . This is wrong in the docs
image

Comment (frenck) [reply source/_integrations/fritz.markdown:59]: Might be, but how is that related to the parent PR of this documentation PR?

Comment (sti0) [reply source/_integrations/fritz.markdown:59]: I thought I fix this issue as I have seen it...

Comment (frenck) [reply source/_integrations/fritz.markdown:59]: Cool, but isn't related to this PR?

Comment (sti0) [reply source/_integrations/fritz.markdown:59]: Actually not to https://github.com/home-assistant/core/pull/62892 if you mean...

Comment (sti0) [reply source/_integrations/fritz.markdown:59]: I will split this into another one tomorrow

**Comment by All112233** [comment]
[]()

**Comment by All112233** [comment]
[](https://github.com/home-assistant/home-assistant.io/commit/938eb98a412d105ff266748cc16a084897234273)

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR. Please git rebase your PR to clean it up.

**Comment by sti0** [comment]
broke something till rebasing...so I started from sketch....https://github.com/home-assistant/home-assistant.io/pull/20915
